src/main/java/org/candlepin/subscriptions/metering/MeteringEventFactory.java

```diff
@@ -131,12 +133,31 @@ public static void updateMetricEvent(
         .withRole(getRole(role, accountNumber, instanceId));
   }
 
-  public static String getEventType(String metricId) {
+  // SWATCH-1374 Remove or update this method
+  public static String getOldEventType(String metricId) {
```

Comment:
We should just keep the same method name but mark this as `Deprecated` show that we'll get compiler and IDE warnings.